api/v1beta1/manila_types.go

```diff
@@ -101,7 +101,7 @@ type ManilaSpec struct {
 
```

Comment:
Wouldn't we also want omitempty on other Optional params? (like this one and several above)